dist/browser.js

```diff
@@ -276,17 +276,17 @@ module.exports = class CalculatorHelpers {
             dataElements.forEach((dataElement) => {
               if (dataElement.id != null) {
                 // if an episode has already been created set the result for the population to 1
-                if (episodeResults[dataElement.id]) {
-                  episodeResults[dataElement.id][popCode] = 1;
+                if (episodeResults[dataElement.id.value]) {
```

Comment:
does it make sense to introduce a local id variable? the same expression is used a couple of times

Reply:
It's just accessing the object property. doesn't seem to be expensive?